aerogearcatalog/aerogear-digger-apb
# xcode-plugin support

Adds xcode-plugin support

**Verification**

* Provision the Mobile CI|CD service
* Run the following command into the jenkins pod: `ls /var/lib/jenkins/plugins/xcode-*` it should find the xcode plugin folder and plugin hpi file
* Login into jenkins and check if the xcode-plugin is installed in `manage jenkins -> plugins -> installed`
